doc/nrf/links.txt

```diff
@@ -1652,8 +1652,8 @@
 
 .. ### Source: files.nordicsemi.com
 
-.. _`Twister test report for nRF Connect SDK v2.9.0-nRF54H20-1-rc3`: https://files.nordicsemi.com/ui/api/v1/download/contentBrowsing/NCS/external/release/v2.9.0-nrf54h20-1-rc3/test-report/twister_test_results_report.html?isNativeBrowsing=true
-.. _`Hardware test report for nRF Connect SDK v2.9.0-nRF54H20-1-rc3`: https://files.nordicsemi.com/ui/api/v1/download/contentBrowsing/NCS/external/release/v2.9.0-nrf54h20-1-rc3/test-report/common_ncs_test_results_report.html?isNativeBrowsing=true
+.. _`Twister test report for nRF Connect SDK v2.9.0-nRF54H20-1`: https://files.nordicsemi.com/ui/api/v1/download/contentBrowsing/NCS/external/release/v2.9.0-nrf54h20-1/test-report/twister_test_results_report.html?isNativeBrowsing=true
+.. _`Hardware test report for nRF Connect SDK v2.9.0-nRF54H20-1`: https://files.nordicsemi.com/ui/api/v1/download/contentBrowsing/NCS/external/release/v2.9.0-nrf54h20-1/test-report/common_ncs_test_results_report.html?isNativeBrowsing=true
```

Comment:
Does not seem to be available, at least yet.

Reply:
does not depend on this pr, will check with the team

Reply:
This was correct in the tagged version, but here it is wrong. Please correct as suggested.